hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/lookup/HoodieLookupTableReader.java

```diff
@@ -49,15 +54,24 @@ public HoodieLookupTableReader(SerializableSupplier<InputFormat<RowData, ?>> inp
   public void open() throws IOException {
     this.inputFormat = inputFormatSupplier.get();
     inputFormat.configure(conf);
-    InputSplit[] inputSplits = inputFormat.createInputSplits(1);
+    this.inputSplits = Arrays.stream(inputFormat.createInputSplits(1)).collect(Collectors.toList());
```

Comment:
this.inputSplits = Arrays.stream(inputFormat.createInputSplits(1))
                                 .collect(Collectors.toCollection(LinkedList::new));